src/Character.js

```diff
@@ -48,6 +48,7 @@ export default class Character extends React.Component<CharacterProps, {}> {
         return (
             <g
                 className='Character'
+                is='svg-group'
```

Comment:
The "is" attribute is for specifying custom HTML elements:

https://html.spec.whatwg.org/multipage/custom-elements.html#customized-built-in-element

I don't think that this is the right approach for silencing the warning from the test as it may have undesirable consequences for the running application, by triggering the custom element processing within the browser.

In this case, I'd rather just live with the warning, if we can find no other solution.

Reply:
OK, I'll take it out.